src/domains/migration/pages/Migration/Migration.tsx

```diff
@@ -36,7 +52,13 @@ export const Migration = () => {
 			<Page pageTitle={t("MIGRATION.PAGE_MIGRATION.TITLE")} isBackDisabled={true} data-testid="Migration">
 				<MigrationHeader onNewMigration={onNewMigrationHandler} />
 
-				{renderMigrations()}
+				<Section className="mt-4">
+					<MigrationTransactionsTable
+						migrationTransactions={migrations}
+						isCompact={isCompact}
+						onClick={() => console.log("row click")}
```

Comment:
@dated shall we provide the clicked transaction item here, or will it be handled differently?

Reply:
yea that's just a stub for now, i'd say we change it when the proper transaction and status querying is added.